FusionIIIT/applications/academic_procedures/forms.py

```diff
@@ -1,6 +1,5 @@
-from django import forms
-
 from applications.academic_information.models import Course
+from django import forms
```

Comment:
This file is different in the PR and your master I just forked. I dont think you made the PR to the updated version.